.github/workflows/check.yml

```diff
@@ -8,9 +8,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
This stinks a little, but in order for `yarn version check` to work, we need at least the git history from `main` to `HEAD`. I'd much rather _only_ fetch exactly those commits (the default behavior is to fetch only a single commit, the branch tip), but I have not yet figured out a good way to do that :P. If anyone has any ideas, lemme know!

Reply:
🤷🏻 I wouldn't stress about it for now. I don't expect the change history here will get huge or have huge binaries or anything.